napari_organoid_counter/_widget.py

```diff
@@ -112,9 +120,98 @@ def __init__(self,
         self.viewer.layers.events.inserted.connect(self._added_layer)
         self.viewer.layers.events.removed.connect(self._removed_layer)
         self.viewer.layers.selection.events.changed.connect(self._sel_layer_changed)
+    
         # setup flags used for changing slider and text of min diameter and confidence threshold
         self.diameter_slider_changed = False 
-        self.confidence_slider_changed = False 
+        self.confidence_slider_changed = False
+
+        # Key binding to return the ID of the bounding boxes selected from the GUI
+        @self.viewer.bind_key('s')
```

Comment:
This function was used for debugging, we can remove it from the final version of the tool.